src/js/pebble-js-app.js

```diff
@@ -103,7 +103,33 @@ function refresh_access_token(refresh_token, code) {
 function do_google_api() {
     use_access_token(function(access_token) {
         // Use access token to make request to Calendar API
-    });
+        var req = new XMLHttpRequest();
+        req.open("GET", "https://www.googleapis.com/calendar/v3/users/me/calendarList?access_token=" + access_token, true);
+        req.onload = function(e) {
+            if (req.readyState == 4 && req.status == 200) {
+                // Get list of Calendars https://developers.google.com/google-apps/calendar/v3/reference/calendarList/list
+                var result = JSON.parse(req.responseText);
+                // Get Calendars https://developers.google.com/google-apps/calendar/v3/reference/calendarList#resource
+                var calenders = result.items;
+                if (!calendars || calendars.length === 0) {
+                    return;
+                }
+                req = new XMLHttpRequest():
```

Comment:
this section onwards should be in a separate function, don't use the same variable names